tests/modules/nf-core/paragraph/multigrmpy/test.yml

```diff
@@ -7,3 +7,13 @@
     - path: output/paragraph/test.json.gz
     - path: output/paragraph/test.vcf.gz
     - path: output/paragraph/versions.yml
+
+- name: paragraph multigrmpy test_paragraph_multigrmpy_empty_vcf
+  command: nextflow run ./tests/modules/nf-core/paragraph/multigrmpy -entry test_paragraph_multigrmpy_empty_vcf -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/paragraph/multigrmpy/nextflow.config
+  tags:
+    - paragraph/multigrmpy
+    - paragraph
+  files:
+    - path: output/paragraph/test.vcf.gz
+      md5sum: 613c5411e5595ba0d4c335499a028305
+    - path: output/paragraph/versions.yml
```

Comment:
The changes look very good. Just a question, does versions.yml need a md5sum?

Reply:
No versions.yml does not need an md5sum, we decided to remove it a couple of months ago :)